sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -466,14 +467,27 @@ public void StartAsyncValidatesProcessExceptionAsync()
         [Test]
         public async Task StartAsyncStartsTheEventProcessorWhenProcessingHandlerPropertiesAreSet()
```

Comment:
Exposing inner errors when calling `StopProcessingAsync` made this test to fail. It was getting a `NullPointerException` when trying to retrieve the partition ids list using an inner consumer.

Reply:
Not sure that I'm following....

Reply:
My bad.

https://github.com/Azure/azure-sdk-for-net/blob/707fe7f2e069fe72ee82814712941989d9d2f9dd/sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs#L658-L661

Before this PR, we were swallowing load balancing exceptions in `StopProcessingAsync` instead of surfacing them to the user. This test started to fail after this was changed.

We were getting a `NullPointerException` because we were calling `GetPartitionIds` from a mock consumer, getting a `null` value instead of an empty array. Just fixed that.

Reply:
ahhhh....   gotcha